README.md

```diff
@@ -316,6 +316,22 @@ If you want to customise these component views, you can [publish the views](#ins
 
 You can use the SEO library of your preference to render the SEO tags on the page, e.g. with [`artesaos/seotools`](https://github.com/artesaos/seotools).
 
+### 5. Configure tailwind
+
+To ensure that tailwind will pick up the css classes used by the flexible content blocks components ...
```

Comment:
@bverbist Is the sentence finished? ;-) I would append ", follow the steps below:" or something.